en/micro-integrator/docs/references/connectors/kafka-connector/kafka-connector-producer-example.md

```diff
@@ -0,0 +1,114 @@
+# Kafka Connector Example
+
+Given below is a sample scenario that demonstrates how to send messages to a Kafka broker via Kafka topics.
+
+## What you'll build
+The publishMessages operation allows you to publish messages to the Kafka brokers via Kafka topics.
+
+Given below is a sample proxy service that illustrates how you can connect to a Kakfa broker with the init operation and then use the publishMessages operation to publish messages via the topic.
```

Comment:
```suggestion
Given below is a sample proxy service that illustrates how you can connect to a Kakfa broker with the `init` operation and then use the `publishMessages` operation to publish messages via the topic.
```

Use single backticks for any operation names, config names, etc.

Reply:
Done.